nekoyume/Assets/_Scripts/UI/Widget/Popup/BuffBonusPopup.cs

```diff
@@ -151,5 +173,39 @@ private void PushAction(bool advanced)
             ActionManager.Instance.HackAndSlashRandomBuff(advanced).Subscribe();
             Close();
         }
+
+        private void ShowBuffListView()
+        {
+            buffListView.SetActive(true);
+            UpdateBuffListView();
+        }
+
+        private void UpdateBuffListView()
+        {
+            var randomBuffsheet = Game.Game.instance.TableSheets.CrystalRandomBuffSheet;
+            var randomBuffGroupsByRank = randomBuffsheet.Select(line => line.Value).GroupBy(row => row.Rank);
+
+            var cellList = cellContainer.GetComponentsInChildren<BuffBonusTitleCell>().Select(cell => cell.gameObject)
+                .Concat(cellContainer.GetComponentsInChildren<BuffBonusBuffCell>().Select(cell => cell.gameObject));
+
+            foreach (var cell in cellList)
+            {
+                Destroy(cell);
+            }
+
+            foreach (var randomBuffGroup in randomBuffGroupsByRank)
+            {
+                // Grade
+                var cell = Instantiate(titlePrefab, cellContainer);
+                cell.GetComponent<BuffBonusTitleCell>().Set(randomBuffGroup.Key);
+
+                foreach (var randomBuffRow in randomBuffGroup)
+                {
+                    // buff
+                    cell = Instantiate(buffPrefab, cellContainer);
```

Comment:
If you cached it when you Instantiate a cell, you can destroy it without calling GetComponentsInChildren()